spec/system/manage_maps_spec.rb

```diff
@@ -51,8 +51,8 @@
     find(".campaign-map-selector", text: "Dwarven Excavation").click
     click_and_move_map(map, from: { x: 300, y: 300 }, to: { x: 50, y: 50 })
 
-    expect(page).to have_map_with_data(map, "x", "500")
-    expect(page).to have_map_with_data(map, "y", "500")
+    expect(page).to have_map_with_data(map, "x", "550")
```

Comment:
I'm not sure why this changed?

Reply:
I had to ensure that maps created by the factories were centered: https://github.com/jdbann/table/pull/7/files#diff-b709bad60390eff76d8e893b0e31911dR18